DragDropUI/DD+View.swift

```diff
@@ -36,10 +36,11 @@ public extension DDProtocol where Self: UIView {
         panGesture.handler = { gesture in
             self.handlePan(panGesture: gesture as! UIPanGestureRecognizer)
         }
-
+        
```

Comment:
Please don't leave 4 empty spaces in empty lines.

Reply:
Ok, done!